### PR TITLE
Prevents fungal TB spores from being synthesized

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
tweak: Fungal tuberculosis spores can no longer be synthesized by machinery. 
/:cl:

I've noticed that most "special" round ending chems like xenomicrobes have the can_synth 0 flag, but fungal TB juice doesn't.

This is mostly to prevent people from getting a nice warm cup of fungal TB from the #35188 SCP-294 ruin.

### CODE REVIEW

**Comment by uraniummeltdown** [comment]
No, fuck this unfun var

**Comment by Time-Green** [comment]
What's the point if you can't get fun chems?

**Comment by 81Denton** [comment]
In what alternate reality is fungal TB fun

**Comment by Time-Green** [comment]
Instant death chems are fun

**Comment by improvedname** [comment]
We do not use ones or zeros please use TRUE or FALSE

**Comment by AlexTheSysop42** [comment]
Doesn't tb just make it so you can't do anything?

Also can_synth = 0 is the way it currently is in all instances

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1104,6 +1104,7 @@
 	id = "fungalspores"
 	description = "Active fungal spores."
 	color = "#92D17D" // rgb: 146, 209, 125
+	can_synth = 0
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: FALSE

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]

**Comment by 81Denton** [comment]
Please explain why I should use FALSE if all other instances use a 0

**Comment by 81Denton** [comment]
Actually let me use FALSE, why the hell was that a 0 in the first place

**Comment by davethwave** [comment]
Meh, viro can make stronger stuffs

**Comment by epochayur** [comment]
are people actually defending TB? what's fun about a disease that's nearly impossible to deal with by the time you realize you're infected?

**Comment by AdamElTablawy** [comment]
about as much as fun as being shot by a deathmix syringe that gibs you almost immediately

not that fun for the victim

fucking hilarious for the guy using it

and nobody wants to be the guy who removes it because they want to use it

**Pull request merged**